app/controllers/emails_controller.rb

```diff
@@ -1,11 +1,87 @@
 class EmailsController < ApplicationController
+
+  def show
+    authorize! :send_email, Email
+    @event = Event.find(params[:event_id])
+
+    @templates = get_templates
+    @addresses = @event.email_addresses_of_type(get_status)
+
+    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses,
+                       subject: '', content: '')
+    render :email
+  end
+
+  def submit
```

Comment:
the problem with this is that you cannot have a second button for the same form that directs to a different function. You can only use "commit" parameter to distinguish the different buttons